spellbook/data_formatting/serialize.py

```diff
@@ -14,9 +14,11 @@ def nested_set(dic, keys, value):
     dic[keys[-1]] = value
 
 
-def maybe_numeric_or_bool(string):
-    if string.lower() == "false" or string.lower() == "true":
-        return bool(string)
+def convert_string(string):
+    if string.lower() == "false":
```

Comment:
Might benefit from a 1 line docstring (any of the functions in this module) - module docstring indicates all these are used to parse CLI input, but it's nice to know expected inputs/outputs, e.g. what CL flags would go through a nested dict vs a nested set.